### PR TITLE
PP-5297: Return payment provider in create mandate response

### PR DESCRIPTION
**Pull request by oswaldquek**

### CODE REVIEW

File: src/main/java/uk/gov/pay/directdebit/mandate/api/CreateMandateResponse.java [hunk 1]
```diff
@@ -45,6 +46,9 @@
 
     @JsonProperty("description")
     private String description;
+    
+    @JsonProperty("payment_provider")
+    private String paymentProvider = PaymentProvider.GOCARDLESS.toString().toLowerCase();
```
Comment (stephencdaly) [inline src/main/java/uk/gov/pay/directdebit/mandate/api/CreateMandateResponse.java]: Can't we get the actual value from the gateway account when we create this response?

Comment (oswaldquek) [reply src/main/java/uk/gov/pay/directdebit/mandate/api/CreateMandateResponse.java]: We could but we don't really have to. it's TDD and as long as the test passes we're good. There isn't going to be another direct debit payment provider anytime soon.

Comment (alexbishop1) [reply src/main/java/uk/gov/pay/directdebit/mandate/api/CreateMandateResponse.java]: We do have another provider: sandbox.

Comment (oswaldquek) [reply src/main/java/uk/gov/pay/directdebit/mandate/api/CreateMandateResponse.java]: sorry, you're right. i've fixed this now.

**Pull request merged**